snappy/snapp_test.go

```diff
@@ -779,10 +780,7 @@ architectures:
 `
 
 	snapPkg := makeTestSnapPackage(c, packageHello)
-	part, err := NewSnapFile(snapPkg, "origin", true)
-	c.Assert(err, IsNil)
-
-	_, err = part.Install(&MockProgressMeter{}, 0)
+	_, err := (&Overlord{}).Install(snapPkg, "origin", 0, &MockProgressMeter{})
```

Comment:
Should this be using the field added in s?  This might be a better approach as we want to keep relevant state around, but hard to say now given the upcoming changes in the state engine. Whatever you find more convenient immediately (just drop the field if not in use, please).